.eslintrc.js

```diff
@@ -28,12 +28,14 @@ module.exports = {
     'import/no-unresolved': 'error',
     'import/named': 'error',
     'prettier/prettier': ['error', { singleQuote: true, trailingComma: 'es5' }],
+    'react/button-has-type': 'error',
```

Comment:
All our buttons aren't in a `<form>` so we never had issues. Still I think this is a good idea, what do you think?

Reply:
Sure.